system/CodeIgniter.php

```diff
@@ -773,6 +813,8 @@ public function displayPerformanceMetrics(string $output): string
      */
     protected function tryToRouteIt(?RouteCollectionInterface $routes = null)
     {
+        $this->benchmark->start('routing');
```

Comment:
I don't disagree with this move but it seems more like an opinion, and it could affect metrics between versions. Maybe just note in the docs what's now routing versus bootstrap?

Reply:
Added in the changelog.